Makefile

```diff
@@ -1,10 +1,17 @@
-.PHONY: depend up down
+.PHONY: depend set-region up down
```

Comment:
Is setting the region is related to the PR title? Maybe you want to split the two into two PRs.

Reply:
I think it is because this PR aims to add some requirements, and actually I changed the commit description to fit this change too. I forgot to change the PR title - will do that now

Reply:
Let me know if you think that is OK now :)

Reply:
It's a good practice to keep PRs atomic. One commit per PR which is related to one atomic change. Are the README changes and the region settings atomic together or each change is atomic by itself?

Reply:
You tell me :-)

Reply:
then yes :)